### PR TITLE
MMCA-4962 Map correct url for cash account notification deletion

### PR DESCRIPTION
**Pull request by harshare**

### CODE REVIEW

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -84,8 +84,12 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
   def sessionCacheUrl(sessionId: String, linkId: String): String =
     customsFinancialsSessionCacheUrl + s"/account-link/$sessionId/$linkId"
 
-  def deleteNotificationUrl(fileRole: FileRole, eori: String): String =
-    customsFinancialsApi + s"/eori/$eori/requested-notifications/$fileRole"
+  def deleteNotificationUrl(fileRole: FileRole, eori: String): String = {
+    fileRole match {
+      case CashStatement => customsFinancialsApi + s"/eori/$eori/notifications/$fileRole"
```
Comment (tyagikumarvineet) [inline app/config/FrontendAppConfig.scala:89]: Could you please make use of interpolation here and line 90 as well?

File: test/config/FrontendAppConfigSpec.scala [hunk 2]
```diff
@@ -67,6 +67,18 @@ class FrontendAppConfigSpec extends SpecBase {
     }
   }
 
+  "deleteNotificationUrl" should {
+    "return correct url for cash account filerole" in new Setup {
```
Comment (tyagikumarvineet) [inline test/config/FrontendAppConfigSpec.scala:71]: better to use FileRole or fileRole. Same for line 76. Also would be great if we could add tests for DutyDefermentStatement, SecurityStatement and PostponedVATStatement. 

Leave the additional tests if it takes time but should not take much time

File: app/config/FrontendAppConfig.scala (hunk 1 above)
Comment (harshare) [reply app/config/FrontendAppConfig.scala:89]: @tyagikumarvineet updated line 89 and 90 to use interpolation - s"$customsFinancialsApi/eori/$eori/notifications/$fileRole"

File: test/config/FrontendAppConfigSpec.scala (hunk 2 above)
Comment (harshare) [reply test/config/FrontendAppConfigSpec.scala:71]: Updated test description to use FileRole and added tests for other statement types

**Review by tyagikumarvineet — APPROVED** [review]

**Review by AdamCoops — APPROVED** [review]

**Pull request merged**